tests/test_openeo_gfmap/test_s1_fetchers.py

```diff
@@ -29,6 +29,8 @@
 from .test_s2_fetchers import POINT_EXTRACTION_DF, test_backends, test_configurations
 
 
+# TODO; remove class. Defining tests within classes lowers the readibility
```

Comment:
I don't agree with this statement. as usualy you would group tests in files that correspond to the source code, you might want to use classes to still have some grouping of tests. (not blocking for this merge)

Reply:
sure!